helm/polaris/values.yaml

```diff
@@ -233,170 +326,288 @@ extraEnv:
-
-    # Minimum number of thread to keep alive.
-    minThreads: 10
-    applicationConnectors:
-      # HTTP-specific options.
-      - type: http
-
-        # The port on which the HTTP server listens for service requests.
-        port: 8181
-
-    adminConnectors:
-      - type: http
-        port: 8182
-
-    # The hostname of the interface to which the HTTP server socket wil be found. If omitted, the
-    # socket will listen on all interfaces.
-    # bindHost: localhost
-
-    #  ssl:
-    #    keyStore: ./example.keystore
-    #    keyStorePassword: example
-    #
-    #    keyStoreType: JKS # (optional, JKS is default)
-
-    # HTTP request log settings
-    requestLog:
-      appenders:
-        # Settings for logging to stdout.
-        - type: console
-
-        # # Settings for logging to a file.
-        # - type: file
-
-        #   # The file to which statements will be logged.
-        #   currentLogFilename: ./logs/request.log
-
-        #   #  When the log file rolls over, the file will be archived to requests-2012-03-15.log.gz,
-        #   # requests.log will be truncated, and new statements written to it.
-        #   archivedLogFilenamePattern: ./logs/requests-%d.log.gz
-
-        #   # The maximum number of log files to archive.
-        #   archivedFileCount: 14
-
-        #   # Enable archiving if the request log entries go to the their own file
-        #   archive: true
-
-  featureConfiguration:
-    ENFORCE_PRINCIPAL_CREDENTIAL_ROTATION_REQUIRED_CHECKING: false
-    SUPPORTED_CATALOG_STORAGE_TYPES:
-      - S3
-      - GCS
-      - AZURE
-      - FILE
-
-  callContextResolver:
+# -- Extra volumes to add to the polaris pod. See https://kubernetes.io/docs/concepts/storage/volumes/.
+extraVolumes: []
+  # - name: extra-volume
+  #   emptyDir: {}
+
+# -- Extra volume mounts to add to the polaris container. See https://kubernetes.io/docs/concepts/storage/volumes/.
+extraVolumeMounts: []
+  # - name: extra-volume
+  #   mountPath: /usr/share/extra-volume
+
+# -- Add additional init containers to the polaris pod(s) See https://kubernetes.io/docs/concepts/workloads/pods/init-containers/.
+extraInitContainers: []
+  # - name: your-image-name
+  #   image: your-image
+  #   imagePullPolicy: Always
+  #   command: ['sh', '-c', 'echo "hello world"']
+
+# -- Configures whether to enable the bootstrap metastore manager job.
+# It is recommended to bootstrap realms using the Polaris Admin Tool; use this only if you need to
+# bootstrap realms while deploying Polaris at the same time.
+bootstrap:
+  # -- Specifies whether the bootstrap metastore manager job should be enabled.
+  enabled: false
+  # -- The names of the realms to bootstrap.
+  realms: []
+  # -- The root credentials to create during the bootstrap. If you don't provide credentials for the
+  # root principal of each realm to bootstrap, random credentials will be generated.
+  # Each entry in the array must be of the form: realm,clientId,clientSecret
+  credentials: []
```

Comment:
This is admittedly not great. I opened https://github.com/apache/polaris/issues/878 to improve this and use secrets instead. But for now we need to stick with credentials in clear text.